src/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocket.cs

```diff
@@ -156,7 +156,7 @@ public Task ConnectAsync(Uri uri, CancellationToken cancellationToken)
 
         private Task ConnectAsyncCore(Uri uri, CancellationToken cancellationToken)
         {
-            _innerWebSocket = new WinHttpWebSocket();
+            _innerWebSocket = WebSocketHandle.Create();
 
```

Comment:
Noted, but .NET Native-related changes are out of scope for this PR: the goal here is simply to create an implementation of ClientWebSocket for *nix platforms that does nothing but throw PNSE.